### ISSUE TITLE
HAVE_CURL always seems to be false

### ISSUE THREAD

**Issue by asgrim** [scoutapp/scout-apm-php-ext#65]
We have `MINFO` as follows:

```c
#if HAVE_CURL
  #if HAVE_SCOUT_CURL
	php_info_print_table_row(2, "curl functions", "Yes");
  #else
	php_info_print_table_row(2, "curl functions", "Not instrumented");
  #endif
#else
	php_info_print_table_row(2, "curl functions", "No");
#endif
```

However, `HAVE_CURL` always seems to be evaluating false (I'm using `libcurl4-openssl-dev`), which means `curl functions` shows as `No` even though `HAVE_SCOUT_CURL` may (or may not) have evaluated to true. curl headers are definitely found (I see `checking for libcurl headers... found with pkg-config` in `./configure`) and therefore curl functions ARE actually instrumented, even though it says `No`.

First question: why is `HAVE_CURL` false, even though the headers (and `libcurl4` itself) is installed?

Second question: do we even need to check `HAVE_CURL`? Since we find headers, surely it's a reasonable assumption `libcurl4` is installed anyway? Can we just assume it's there?

**Comment by asgrim** [comment]
Note; as a workaround, customers with the issue can carefully check the `./configure` output.

The following output indicates curl headers are present, and `scoutapm` will instrument curl calls:

```
checking for pkg-config... /usr/bin/pkg-config
checking for curl-config... /usr/bin/curl-config
checking for libcurl headers... found with pkg-config
```

The following output indicates curl headers are NOT present, and `scoutapm` will NOT instrument curl calls:

```
checking for pkg-config... /usr/bin/pkg-config
checking for curl-config... no
checking for libcurl headers... configure: WARNING: neither pkg-config nor curl-config found
configure: WARNING: curl library headers were not found on the system, scoutapm will not instrument curl functions
```

**Comment by asgrim** [comment]
@cschneid did [some digging](https://github.com/php/php-src/blob/85b5dc4711f6ce8b979ed60e3b60cf93225be32a/ext/curl/config.m4#L77) and I think we've figured out; if PHP is compiled for apt package managers (e.g. I'm using the `deb.sury.org` packages) then because `curl` is a separate extension, then `HAVE_CURL` might always be false. Therefore it might be more sensible to display it separately. We don't actually check `HAVE_CURL` for the main code inclusion (we use `HAVE_SCOUT_CURL` for that) so functionally, it doesn't matter anyway. Before we do this; we need to check exactly what happens if `scoutapm` is compiled WITH curl, but `php7.4-curl` (as a distro package for example) is NOT installed. Obviously, a crash is not acceptable.

**Comment by asgrim** [comment]
Fixed in #126
